### DISCUSSION TITLE
lidR On Windows

### DISCUSSION THREAD

**Discussion by AbdullahJirjees** [r-lidar/lidR#653] [Q&A]
Hello @Jean-Romain 
I am testing your repo but on a Windows environment, I wonder if it is able to run on Windows or I must switch to Ubuntu.
https://github.com/r-lidar/lidR

**Comment by Jean-Romain** [comment]
lidR is an R package and is available on all major plateforms: Windows/Linux/Mac.

**Reply by Jean-Romain** [reply]
In R

```r
install.packages("lidR")
```

**Reply by Jean-Romain** [reply]
Every single function is documented with copy pastable examples. There is also a book https://r-lidar.github.io/lidRbook/

**Reply by Jean-Romain** [reply]
For large collection of files you can use LAScatalog parallelism capabilities https://r-lidar.github.io/lidRbook/engine.html (parallism is in section 14.7.4.

On a single point cloud if the code is not already multicore it means that the algorithm is not parallélisable.

**Reply by Jean-Romain** [reply]
If the algorithm is not parallizable this won't magically make it parallel. And tree segmentation algorithms are not parallelizable.

**Reply by AbdullahJirjees** [reply]
Thanks a lot for clearing that, have a nice weekend.

**Comment by luiz231827** [comment]
For the segmentation of individual trees. I apply it in my point cloud the field of study is the jungle in tucatan mexico where the jungle predominates. How to know that they
